.travis.yml

```diff
@@ -58,8 +58,8 @@ script:
 - gradle test -b test/java/build.gradle
 - go test -coverprofile=coverage.txt -covermode=atomic github.com/anz-bank/sysl/...
 - npm test --prefix sysl2/sysl/sysl_js
-- GOOS=darwin GOARCH=amd64 go build -o gosysl/gosysl-darwin github.com/anz-bank/sysl/sysl2/sysl
-- GOOS=linux GOARCH=amd64 go build -o gosysl/gosysl-linux github.com/anz-bank/sysl/sysl2/sysl
+- GOOS=darwin GOARCH=amd64 go build -o gosysl/gosysl-darwin -ldflags "-X main.Version=${TRAVIS_TAG:1}" github.com/anz-bank/sysl/sysl2/sysl
+- GOOS=linux GOARCH=amd64 go build -o gosysl/gosysl-linux -ldflags "-X main.Version=${TRAVIS_TAG:1}" github.com/anz-bank/sysl/sysl2/sysl
```

Comment:
Defaulting to 1 doesn't look right. Better to use something like "unspecified".

Reply:
It is not defaulting to 1. As git tag it defined like `v0.3.0`, just removing `v` from tag.

Reply:
Ah, silly me. I mistook `:` for `:-`.